modules/ROOT/pages/microprofile-telemetry.adoc

```diff
@@ -44,13 +85,31 @@ otel.traces.exporter=zipkin
 otel.exporter.zipkin.endpoint=http://localhost:9411/api/v2/spans
 ----
 
+If you want to export traces to OpenLiberty's log files use the following properties:
+
+[source,properties]
+----
+otel.sdk.disabled=false
+otel.traces.exporter=logging
+otel.exporter.zipkin.endpoint=http://localhost:9411/api/v2/spans
```

Comment:
Remove this line. You would not need the Zipkin exporter endpoint if you are exporting to logs